ci-operator/step-registry/ovn/local-to-shared-gateway-mode-migration/ovn-local-to-shared-gateway-mode-migration-commands.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+echo "Changing to shared gateway mode"
+echo "-------------------"
+oc patch Network.operator.openshift.io cluster --type='merge' --patch "{\"spec\":{\"defaultNetwork\":{\"ovnKubernetesConfig\":{\"gatewayConfig\":{\"routingViaHost\":false}}}}}"
+
+oc wait co network --for='condition=PROGRESSING=True' --timeout=30s
+# Wait until the ovn-kubernetes pods are restarted
+timeout 300s oc rollout status ds/ovnkube-master -n openshift-ovn-kubernetes
+timeout 300s oc rollout status ds/ovnkube-node -n openshift-ovn-kubernetes
```

Comment:
can you craft some command that will validate the gateway mode on the running cluster? I remember doing it with an 'oc logs' command and looking for the string that configures it. then if it's wrong we can exit 1 and the job will quit and help us catch any bug where the `oc patch` command doesn't do what we think.

Reply:
we can just check `oc describe Network.operator.openshift.io cluster` plus the logs in CI show if the changed worked: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/24692/rehearse-24692-pull-ci-openshift-ovn-kubernetes-master-e2e-aws-ovn-shared-to-local-gateway-mode-migration/1471495402858483712/artifacts/e2e-aws-ovn-shared-to-local-gateway-mode-migration/ovn-shared-to-local-gateway-mode-migration/build-log.txt